modules/apps/object/object-test/src/testFunctional/tests/Picklist.testcase

```diff
@@ -1021,10 +1021,10 @@ definition {
 			ObjectAdmin.selectDropdownItem(
 				currentOption = "Select an option",
 				labelName = "Tipo",
-				optionName = "Lista de seleção");
+				optionName = "Picklist");
```

Comment:
@andreefarias is this correct expected behavior? The option name will no longer be translated?

Reply:
Yes, should be.